gatewayservice/gateway-service.js

```diff
@@ -41,6 +42,15 @@ app.post('/adduser', async (req, res) => {
   }
 });
 
+app.get('/questions', async (req, res) => {
```

Comment:
Cambiar la ruta en el resto de sitios acorde a lo nuevo establecido en el endpoint del servicio
```suggestion
app.get('/history/questions', async (req, res) => {
```